### PR TITLE
removed eoscanada and more on eosq.app

### PR DESCRIPTION
**Pull request by phila82r**

### CODE REVIEW

**Comment by dliuproduction** [comment]
Builds and runs locally. Changes are only concerning design.

File: eosq/src/components/footer/footer.component.tsx [hunk 1]
```diff
@@ -77,19 +77,18 @@ const BaseFooter = () => (
               q
             </LogoSecond>
           </LogoLink>
-          <Cell pt={[3]}>
-            <Text color={theme.colors.primary}>{t("footer.subTitle")}</Text>
-          </Cell>
-          <ExternalTextLink to="https://eoscanada.com">
-            <StyledLogoImg src="/images/logo-eos-canada.svg" />
-          </ExternalTextLink>
+          {/* <Cell pt={[3]}>
```
Comment (maoueh) [inline eosq/src/components/footer/footer.component.tsx:80]: @phila82r Don't leave dead code around, please open another PR to remove all these.

File: eosq/src/i18n/zh/footer.i18n.ts [hunk 2]
```diff
@@ -1,15 +1,16 @@
 export default {
   footer: {
-    telegram: "Telegram技术支持群",
-    subTitle: "由 dfuse 为 EOS 社区而开发",
-    copyright: "© {{year}} dfuse Platform Inc. 版权所有	",
-    madeWithLove: "Made with Love by",
+    telegram: "Telegram 社区&团队支持群",
+    subTitle: "need to translate",
```
Comment (maoueh) [inline eosq/src/i18n/zh/footer.i18n.ts:4]: Missing translation.

File: eosq/src/i18n/zh/footer.i18n.ts [hunk 3]
```diff
@@ -1,15 +1,16 @@
...
+    aboutDfuse: "dfuse 是一个开源区块链数据平台",
+    copyright: "© 2018-{{year}} dfuse Platform Inc. 版权所有",
     privacyPolicy: "隐私政策",
-    privacyPolicyLink: "https://www.eoscanada.com/en/privacy-policy",
+    privacyPolicyLink: "https://dfuse.io/zh/%E9%9A%90%E7%A7%81%E6%94%BF%E7%AD%96/",
```
Comment (maoueh) [inline eosq/src/i18n/zh/footer.i18n.ts:8]: On that page, the privacy policy link to old URL. Just wanted to be sure that's an OK change.

File: eosq/src/i18n/zh/footer.i18n.ts [hunk 4]
```diff
@@ -1,15 +1,16 @@
...
     termsOfServices: "服务条款",
-    termsOfServicesLink: "https://www.dfuse.io/en/terms",
-    eoscanada: "EOS Canada",
-    eoscanadaLink: "https://www.eoscanada.com",
-    eoscanadaLinkCombined: "<0>EOS Canada</0> 用爱制造"
+    termsOfServicesLink: "https://dfuse.io/en/terms"
   }
 }
+
+/**
```
Comment (maoueh) [inline eosq/src/i18n/zh/footer.i18n.ts:14]: Weird, what's that?

Comment (maoueh) [reply eosq/src/i18n/zh/footer.i18n.ts:14]: Oh it's the comment that should go on `subTitle` not translated. We could remove the `subTitle` key then and it's `en` counterpart.

File: eosq/src/components/footer/footer.component.tsx (hunk 1 above)
Comment (phila82r) [reply eosq/src/components/footer/footer.component.tsx:80]: I wasn't sure if these were gonna be used again at some point (this is related to "subTitle" not being used), so I kept them there. So what you're saying is that, whatever the case, if some of the code is not used it should be removed entirely?

File: eosq/src/i18n/zh/footer.i18n.ts (hunk 2 above)
Comment (phila82r) [reply eosq/src/i18n/zh/footer.i18n.ts:4]: should be removed entirely

File: eosq/src/components/footer/footer.component.tsx (hunk 1 above)
Comment (maoueh) [reply eosq/src/components/footer/footer.component.tsx:80]: Yes exact, Git is going to be there if we need to go back in time.

File: eosq/src/i18n/zh/footer.i18n.ts (hunk 3 above)
Comment (phila82r) [reply eosq/src/i18n/zh/footer.i18n.ts:8]: That's not an old url... is it? It's the URL in the footer of our Chinese version website at https://www.dfuse.io/zh (see this screenshot: https://share.getcloudapp.com/E0ubbpRb)

The eoscanada.com URL was the old privacy policy link, no?

File: eosq/src/i18n/zh/footer.i18n.ts (hunk 4 above)
Comment (phila82r) [reply eosq/src/i18n/zh/footer.i18n.ts:14]: agreed

File: eosq/src/i18n/zh/footer.i18n.ts (hunk 3 above)
Comment (maoueh) [reply eosq/src/i18n/zh/footer.i18n.ts:8]: You are right, quickly misread the link. My comment is more that the new link does not point directly to privacy policy page (which is now https://www.dfuse.io/en/privacy-policy).

When you click on `https://dfuse.io/zh/%E9%9A%90%E7%A7%81%E6%94%BF%E7%AD%96/`, there is an extract click to see the privacy policy. So just wanted to be sure it was OK.

But you are right that the link needs to change nonetheless to something else.

Comment (phila82r) [reply eosq/src/i18n/zh/footer.i18n.ts:8]: Are you referring to this link? (screenshot: https://share.getcloudapp.com/lluDDKmp) If so, this basically says 

> This document is the Chinese translation of the original, if there is any difference, please refer to the original dfuse "Privacy Policy"

The link's Chinese characters `隐私政策` = `%E9%9A%90%E7%A7%81%E6%94%BF%E7%AD%96` so I believe the url is correct?

**Pull request merged**